pyiron_gui/project/project_browser.py

```diff
@@ -23,6 +29,207 @@
 __date__ = "Feb 02, 2021"
 
 
+class PyironWrapper:
+
+    """Simple wrapper for pyiron objects which extends for basic pyiron functionality (list_nodes ...)"""
+
+    _with_self_representation = {
+        "structure": 'pyiron_atomistics.atomistics.structure.atoms.Atoms',
+        "murnaghan": 'pyiron_atomistics.atomistics.master.murnaghan.Murnaghan'
+    }
+
+    def __init__(self, pyi_obj, project, rel_path=""):
+        self._wrapped_object = pyi_obj
+        self._project = project
+        if not hasattr(pyi_obj, 'path'):
+            self._path = self._project.path
+        self._rel_path = rel_path
+        # print("init:" + self.path)
+        self._type = None
+        for key, value in self._with_self_representation.items():
+            if static_isinstance(pyi_obj, value):
+                self._type = key
+
+    @property
+    def has_self_representation(self):
+        return self._type is not None
+
+    @staticmethod
+    def _empty_list():
+        return []
+
+    @property
+    def name(self):
+        return self._type
+
+    @property
+    def project(self):
+        if hasattr(self._wrapped_object, 'project'):
+            return self._wrapped_object.project
+        return self._project
+
+    @property
+    def path(self):
+        if hasattr(self._wrapped_object, 'path'):
+            return self._wrapped_object.path
+        if hasattr(self.project, 'path'):
+            return self.project.path + '/' + self._rel_path
```

Comment:
My gut says `os.path.join(self.project.path, self._rel_path)` is safer, but I know elsewhere we're always doing `.replace('\\', '/')` (or something like that) for windows machines. Maybe because our HDF path wants `/` regardless of the OS? Anyway, take this advice with a grain of salt.

Reply:
Actually, all our `path` business works with unix paths (`Project('.').path` returns a unix path regardless of the OS). What I could use is probably `posixpath.join`